webdriver/tests/bidi/input/perform_actions/key.py

```diff
@@ -24,8 +24,6 @@ async def test_key_backspace(bidi_session, top_context, setup_key_test):
     [
         ("\U0001F604"),
         ("\U0001F60D"),
-        ("\u0BA8\u0BBF"),
-        ("\u1100\u1161\u11A8"),
```

Comment:
I would leave this review to @jgraham. But if we are going to remove these lines we should remove them as well from the WebDriver classic tests.

Reply:
WebDriver classic doesn't have these test.

Reply:
The tests are at https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/classic/perform_actions/key_special_keys.py#L11-L12.

Reply:
I stand corrected! I was looking at the bottom test.

Interesting this file tests that multiple code points fails and succeeds...